idx: 450
erasor: 
    max_range: 80.0
    num_rings: 20
    num_sectors: 108
    min_h: -0.8 # [m] Note that it depends on the distance between body frame and ground contact point of mobile robots
    max_h: 3.1 # [m] Note that it depends on the distance between body frame and ground contact point of mobile robots
    th_bin_max_h: 0.2 # [m]
    scan_ratio_threshold: 0.20 # The Larger, the more aggressive!!
    minimum_num_pts: 6
    rejection_ratio: 0
    gf_dist_thr: 0.125
    gf_iter: 3
    gf_num_lpr: 10
    gf_th_seeds_height: 0.5
    version: 3 # 2: R-GPF / 3: R-GPF w/ blocking (not in paper)
    num_lowest_pts: 1


MapUpdater:
    data_name: "07"
    initial_map_path: "/media/shapelim/UX960NVMe1/kitti_semantic/semantic_KITTI_map/src_0_2/07_630_to_820_w_interval2_voxel_0.200000.pcd"
    env: "outdoor"
    save_path: "/media/shapelim/UX960NVMe1/kitti_semantic/semantic_KITTI_map/erasor_v2"
    query_voxel_size: 0.2  
    map_voxel_size: 0.2 
    voxelization_interval: 10
    removal_interval: 5

tf:
     lidar2body: [0.0, 0.0, 1.73, 0, 0.0, 0.0, 1.0] # xyz q_x, q_y, q_z, q_w in order

verbose: true


